Cargo.toml

```diff
@@ -19,14 +19,13 @@ members = [
   "examples/js_callback",
   "examples/keyed_list",
   "examples/mount_point",
-  "examples/multi_thread",
+  "examples/agents",
   "examples/nested_list",
   "examples/node_refs",
   "examples/password_strength",
   "examples/portals",
-  "examples/pub_sub",
+  "examples/contexts",
   "examples/router",
```

Comment:
Please sort this alphabetically so it's easier to find members based on the directory structure